strax/context.py

```diff
@@ -1855,11 +1855,24 @@ def copy_to_frontend(self,
                 # Need to load a new loader each time since it's a generator
                 # and will be exhausted otherwise.
                 loader = s_be.loader(s_be_key)
+
+                def wrapped_loader():
+                    """Wrapped loader for changing the target_size_mb"""
+                    while True:
+                        try:
+                            # pylint: disable=cell-var-from-loop
+                            data = next(loader)
+                            # Update target chunk size for re-chunking
+                            data.target_size_mb = md['chunk_target_size_mb']
+                        except StopIteration:
+                            return
+                        yield data
```

Comment:
Apparently this was a bug that I did not notice before - you could only rechunk until the chunk default target mb size